src/index.js

```diff
@@ -77,6 +91,16 @@ function activateMoreListeners() {
   });
 }
```

Comment:
The functions activateMoreListeners, activateCheckboxListeners, activateTaskInputListeners, and activateCompletedListener share similar patterns. Consider creating a generic function to reduce duplication.